Package.swift

```diff
@@ -0,0 +1,62 @@
+// swift-tools-version:5.0
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import PackageDescription
+
+let package = Package(
+    name: "App Center",
+    products: [
+        .library(
+            name: "AppCenter",
+            targets: ["AppCenter"]),
+        .library(
+            name: "AppCenterAnalytics",
+            targets: ["AppCenterAnalytics"])
+    ],
+    dependencies: [
+    ],
+    targets: [
+        .target(
+            name: "AppCenter",
+            dependencies: [],
+            path: "AppCenter/AppCenter",
+            exclude: ["Support"],
+            cSettings: [
+                .define("APP_CENTER_C_NAME", to: "\"appcenter.ios\"", .when(platforms: [.iOS])),
+                .define("APP_CENTER_C_NAME", to: "\"appcenter.macos\"", .when(platforms: [.macOS])),
+                .define("APP_CENTER_C_NAME", to: "\"appcenter.tvos\"", .when(platforms: [.tvOS])),
+                .define("APP_CENTER_C_VERSION", to:"\"3.1.1\""),
```

Comment:
Do we really need extra quotes here?

Reply:
Muotes are mandatory in define values, yes.